### PR TITLE
Don't use prototype extensions.

### PR DESCRIPTION
**Pull request by abuiles**

### CODE REVIEW

File: addon/components/auto-complete.js [hunk 1]
```diff
@@ -5,9 +5,10 @@ export default Ember.Component.extend({
   hightlightIndex: -1,
   escapedChars: [40, 38, 13],
   visibility: "display:none;",
-  inputClazz: function() {
+  inputClass: '',
```
Comment (abuiles) [inline addon/components/auto-complete.js:8]: I saw an undefined in the class all the time, I suppose the reason is that the property wasn't defined.

File: tests/dummy/app/components/my-auto-complete.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
+import Ember from 'ember';
 import AutoComplete from "ember-cli-auto-complete/components/auto-complete";
 
 export default AutoComplete.extend({
   valueProperty: "code",
-  suggestions: function() {
+  suggestions: Ember.computed("inputVal", "options.[]", function() {
```
Comment (abuiles) [inline tests/dummy/app/components/my-auto-complete.js:6]: :trollface:

**Comment by toranb** [comment]
So this is just a basic refactor - correct? I see all the tests are passing --- anything else we need to add?

The only update I'd ask for -would you mind renaming the commit to have the word [REFACTOR]: as a prefix? I'm trying to do a _better_ job than I've done in the past (labeling what I did /etc)

**Comment by abuiles** [comment]
@toranb correct, is a simple refactor. re: tagging, done.

**Comment by toranb** [comment]
Last question - what version of ember and ember-cli do you need to get this new computed right now?

**Comment by abuiles** [comment]
@toranb this doesn't depend on any version of ember-cli. About Ember's I'm almost  sure I've used this as far as back as 1.7, .property and .on are just sugar  ... @stefanpenner can you confirm about the versions please?

**Comment by stefanpenner** [comment]
This syntax has existed since the beginning of time. In-fact all of ember has been written this way.

**Comment by toranb** [comment]
Excellent - thanks for the clarification @stefanpenner

**Comment by toranb** [comment]
@abuiles just did an npm publish of 0.0.11 that includes this change

**Pull request merged**